src/app/login/page.tsx

```diff
@@ -1,35 +1,35 @@
 'use client';
 
 import { useState } from 'react';
+import { useRouter } from 'next/navigation';
 import supabase from '../../api/supabase/createClient';
 
 export default function Login() {
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
+  const { push } = useRouter();
   const handleSignUp = async () => {
-    const { data, error } = await supabase.auth.signUp({
+    const { error } = await supabase.auth.signUp({
       email,
       password,
     });
 
     if (error) {
       throw new Error(`An error occurred trying to sign up: ${error}`);
     }
```

Comment:
As it is right now, if an error occurs, it will log "An error occurred trying to sign in: `[object Object]`", since the `error` variable returned by the Supabase call is an error object. I would recommend to just `throw error` instead, or say `error.message` if we want the context.